HHS/OPRE-OPS
# OPS-1423 Update Portfolio Title Card

## What changed

Portfolio Title Card updated for design changes 
* Update the Team Leader tag to match recent Figma design
* Add the label "Portfolio Description" above the description text

## Issue

#1423 

## How to test

* view the portfolio details page for a portfolio and verify the title card design

## Screenshots
![image](https://github.com/HHS/OPRE-OPS/assets/127965976/f6053105-589a-4f4d-be52-ede3cfa07616)

## Definition of Done Checklist


## Links

* Figma designs here: https://www.figma.com/file/dsQFe6G2KjcTWNq7NNZujm/OPRE-OPS?type=design&node-id=5001-159899&mode=dev
